src/libs/CardUtils.ts

```diff
@@ -158,6 +159,14 @@ function getTranslationKeyForLimitType(limitType: ValueOf<typeof CONST.EXPENSIFY
     }
 }
 
+function getEligibleBankAccountsForCard(bankAccountsList: OnyxEntry<BankAccountList>) {
+    if (!bankAccountsList || isEmptyObject(bankAccountsList)) {
+        return [];
+    }
+    /* eslint-disable @typescript-eslint/prefer-nullish-coalescing */
+    return Object.values(bankAccountsList).filter((bankAccount) => bankAccount?.accountData?.allowDebit || bankAccount?.accountData?.type === CONST.BANK_ACCOUNT.TYPE.BUSINESS);
```

Comment:
if allowDebit would be false it would return false and wouldn't check for the type. We could have false for allowDebit but type would be fine, but we would get false incorrectly

Reply:
Could we get around this? I believe that any eslnt rule disable means that react-compiler does not process this file, we should try to avoid adding any eslint-disable to new code